build_tools/azure/install.sh

```diff
@@ -38,7 +38,10 @@ if [[ "$DISTRIB" == "conda" ]]; then
     if [[ "$UNAMESTR" == "Darwin" ]]; then
         if [[ "$SKLEARN_TEST_NO_OPENMP" != "true" ]]; then
             # on macOS, install an OpenMP-enabled clang/llvm from conda-forge.
-            TO_INSTALL="$TO_INSTALL conda-forge::compilers>=1.0.4 \
+            # TODO: Remove <1.1.0 when the following is fixed:
+            # sklearn/svm/_libsvm.cpython-38-darwin.so,
+            # 2): Symbol not found: _svm_check_parameter error
+            TO_INSTALL="$TO_INSTALL conda-forge::compilers>=1.0.4,<1.1.0 \
```

Comment:
I'm just wondering how we'd remember to fix this :P is there an upstream issue which we can track?

Reply:
Or just add a condition to exclude 1.1.0 release and have it automatically triggered and possibly fail if the next release also has the issue?

Reply:
When we had an issue with `ranlib` we were told to report in `conda-forge/python-feedstock`: https://github.com/conda-forge/python-feedstock/issues/302

It looks like this was the same issue we ran into when building 0.22.1:

https://github.com/conda-forge/scikit-learn-feedstock/pull/115#issuecomment-570242103